sdk/core/scheduler/main.cc

```diff
@@ -258,6 +258,15 @@ namespace sched
 	                                               size_t        mepc,
 	                                               size_t        mtval)
 	{
+#if !defined(NDEBUG)
```

Comment:
We don’t really use these macros anywhere, this should probably be a constexpr if on the scheduler debug flag.